patches/minecraft/net/minecraft/world/gen/ChunkGeneratorOverworld.java.patch

```diff
@@ -43,6 +43,15 @@
          double d0 = 0.03125D;
          this.field_186002_u = this.field_185994_m.func_151599_a(this.field_186002_u, (double)(p_185977_1_ * 16), (double)(p_185977_2_ * 16), 16, 16, 0.0625D, 0.0625D, 1.0D);
  
+@@ -183,7 +205,7 @@
+         this.func_185976_a(p_185932_1_, p_185932_2_, chunkprimer);
+         this.field_185981_C = this.field_185995_n.func_72959_q().func_76933_b(this.field_185981_C, p_185932_1_ * 16, p_185932_2_ * 16, 16, 16);
+         this.func_185977_a(p_185932_1_, p_185932_2_, chunkprimer, this.field_185981_C);
+-
++        net.minecraftforge.event.ForgeEventFactory.onChunkGenerate(this, this.field_185995_n, p_185932_1_, p_185932_2_, chunkprimer);
```

Comment:
Don't remove the blank line.

Reply:
it's more of a "don't change more lines then needed" afaik, if there is already a blank line leave it, if there isn't then don't add any extra around your code

Reply:
Ok, I think I have misinterpreted that before